pinot-hadoop/src/main/java/org/apache/pinot/hadoop/job/SegmentPreprocessingJob.java

```diff
@@ -404,9 +278,9 @@ private void fetchResizingConfig() {
       return;
     }
     Map<String, String> customConfigsMap = tableCustomConfig.getCustomConfigs();
-    if (customConfigsMap != null && customConfigsMap.containsKey("min.num.output.files")) {
-      _numberOfOutputFiles = Integer.parseInt(customConfigsMap.get("min.num.output.files"));
-      Preconditions.checkArgument(_numberOfOutputFiles > 0, String.format("The value of min.num.output.files should be positive! Current value: %s", customConfigsMap.get("min.num.output.files")));
+    if (customConfigsMap != null && customConfigsMap.containsKey(InternalConfigConstants.PREPROCESS_NUM_FILES)) {
+      _numberOfOutputFiles = Integer.parseInt(customConfigsMap.get(InternalConfigConstants.PREPROCESS_NUM_FILES));
+      Preconditions.checkArgument(_numberOfOutputFiles > 0, String.format("The value of %s should be positive! Current value: %s", InternalConfigConstants.PREPROCESS_NUM_FILES, customConfigsMap.get(InternalConfigConstants.PREPROCESS_NUM_FILES)));
```

Comment:
(nit) Second argument can be replaced with `_numberOfOutputFiles`, same for line 297